packages/relay/src/lib/eth.ts

```diff
@@ -666,57 +752,41 @@ export class EthImpl implements Eth {
                 let balanceFromTxs = 0;
                 if (mirrorAccount.balance) {
                   currentBalance = mirrorAccount.balance.balance;
-                  currentTimestamp = mirrorAccount.balance.timestamp;
-                }
-
-                // Need to check if there are any transactions before the block.timestamp.to in the current account set returned from the inital
-                // call to getAccountPageLimit.  If there are we may need to paginate.
-                let lastTransactionOnPageTimestamp;
-                if(mirrorAccount.links.next !== null) {
-                  // Get the end of the page of transactions timestamp
-                  const params = new URLSearchParams(mirrorAccount.links.next.split('?')[1]);
-                  if((params === null) || (params === undefined)) {
-                    this.logger.debug(`${requestIdPrefix} Unable to find expected search parameters in account next page link ${mirrorAccount.links.next}), returning 0x0 balance`);
-                    return EthImpl.zeroHex;
-                  }
-
-                  const timestampParameters = params.getAll('timestamp'); 
-                  lastTransactionOnPageTimestamp = timestampParameters[0].split(':')[1];
-                  if((lastTransactionOnPageTimestamp === null) || (lastTransactionOnPageTimestamp === undefined)) {
-                    this.logger.debug(`${requestIdPrefix} Unable to find expected beginning (gte:) timestamp in account next page link ${mirrorAccount.links.next}), returning 0x0 balance`);
-                    return EthImpl.zeroHex;
-                  }
                 }
 
-                let transactionsInTimeWindow: any = [];
-                if((typeof lastTransactionOnPageTimestamp !== "undefined") && (mirrorAccount.transactions[mirrorAccount.transactions.length -1].consensus_timestamp >= lastTransactionOnPageTimestamp)) {
-                  transactionsInTimeWindow = await this.mirrorNodeClient.getTransactionsForAccount(
-                    mirrorAccount.account,
-                    block.timestamp.to,
-                    currentTimestamp,
-                    requestId
-                  );
-                } else {
-                    transactionsInTimeWindow = mirrorAccount.transactions.filter((tx: any) => {
-                    return tx.consensus_timestamp >= block.timestamp.to && tx.consensus_timestamp <= currentTimestamp;
-                  });
-                }
-
-                for(const tx of transactionsInTimeWindow) {
-                  for (const transfer of tx.transfers) {
-                    if (transfer.account === mirrorAccount.account && !transfer.is_approval) {
-                      balanceFromTxs += transfer.amount;
-                    }
+                // The balance in the account is real time, so we simply subtract the transactions to the block.timestamp.to to get a block relevant balance.
```

Comment:
Yes, so my balance result change is to the the point in time we use to consider to get the balance.  I changed it from the block.timestamp.from to the block.timestamp.to as it marks the end of a block.  The old code had the following:
```
const blockTimestamp = Number(block.timestamp.from.split('.')[0]);
const timeDiff = latestTimestamp - blockTimestamp;
```
Now to answer your question, yes I reverted back to the old order of operations, that was to add up all the transfers from the block.timestamp.to, and subtract from the balance at the latest block, instead of subtract all of the transfers up to the block.timestamp.to.  The result is the same and I felt as if it followed the original logic more, so would be less confusing to anyone reviewing this.